spec/unit/tab_spec.rb

```diff
@@ -3,12 +3,12 @@
   before do
     @app = TestDelegate.new
 
-    @tab_1 = TabScreen.new
-    @tab_2 = BasicScreen.new
-    @tab_3 = HomeScreen.new
-    @tab_4 = TestTableScreen.new
+    @tab_0 = TabScreen.new
+    @tab_1 = BasicScreen.new
+    @tab_2 = HomeScreen.new
+    @tab_3 = TestTableScreen.new
```

Comment:
lol... had to make sure we were zero-indexed for that array so it would match the indexes stored in persistence for easy visual comparison of the tests.